nixos/modules/services/misc/bees.nix

```diff
@@ -0,0 +1,94 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  inherit (lib) mkOption types optionalString;
```

Comment:
Can remove this line, the `with` above includes all those in scope automatically.